src/views/v2/widget-list.php

```diff
@@ -0,0 +1,8 @@
+<?php
+/*
+ * Temporary Display of V2 List Widget
```

Comment:
:no_entry_sign: You must use "/**" style comments for a file comment

<sub>hash: 998d08cb4230de258301d45335036615</sub>